cni/netconfig.go

```diff
@@ -28,9 +28,16 @@ type PortMapping struct {
 	Protocol      string `json:"protocol"`
 	HostIp        string `json:"hostIP,omitempty"`
 }
-
 type RuntimeConfig struct {
-	PortMappings []PortMapping `json:"portMappings,omitempty"`
+	PortMappings []PortMapping    `json:"portMappings,omitempty"`
+	DNS          RuntimeDNSConfig `json:"dns,omitempty"`
+}
+
+// https://github.com/kubernetes/kubernetes/blob/7dbe426b2426458f02339e28d1cc102efdb6eda1/pkg/kubelet/dockershim/network/cni/cni.go#L104
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/network/cni/cni.go#L104

Reply:
@ashvindeodhar cniDNSConfig is not exported..so I don't think it can be used